CheckerUI_xforms/CheckerUI/CheckerUI/CheckerUI/Helpers/Order/OrdersManager.cs

```diff
@@ -0,0 +1,144 @@
+            //CreateSignal();
+            //m_ToDoList = new ObservableCollection<ToDo>();
+            //App.HubConn.On<String, DateTime>("ReceiveList", (desc, date) =>
+            //{
+            //    ToDo todo = new ToDo() { createdDate = date, description = desc };
+            //    m_ToDoList.Add(todo);
+            //});
+            generateOrders(15);
+            //  updateLines();
+            //foreach (var item in m_OrdersList[0].m_Items)
+            //{
+            //    m_OrderItemModelList.Add(item);
+            //}
+            //m_ItemsViewManager = new OrderItemViewManager(m_OrderItemModelList);
+        }
+
+        public void UpdateLines(ObservableCollection<OrderItemView> i_OrderItems)
+        {
+         
+            foreach (var item in i_OrderItems)
+            {
+                itemsLineView.Add(item);
+            }
+        }
+
+        public void UpdateAllLinesByOrders()
+        {
+            foreach (var order in m_OrdersViews)
+            {
+                UpdateLines(order.Items);
+            }
+        }
+        public async void CreateSignal()
+        {
+            if (App.HubConn.State == HubConnectionState.Disconnected)
+            {
+                try
+                {
+                    await App.HubConn.StartAsync();     // start async connection to SignalR Hub at server
+                    await App.HubConn.InvokeAsync("InitialToDos");  // invoke initial event - to get all current listings
+                }
+                catch (System.Exception ex)
+                {
+                    Console.WriteLine(ex.Message);
+                }
+            }
+        }
+
+        private void generateOrders(int i_Size)
+        {
+            for (int i = 0; i < i_Size; i++)
+            {
+                OrderModel model = generateOrderModel();
+                OrderViewModel viewModel = new OrderViewModel(model);
+                viewModel.AllItemsCheckedOrderID.CollectionChanged += AllItemsCheckedOnCollectionChanged;
+                m_Orders.Add(model.m_OrderID, viewModel);
+                m_OrdersViews.Add(viewModel);
+            }
+        }
+
+        private void AllItemsCheckedOnCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
+        {
+            var observer = sender as ObservableCollection<int>;
+            var id = observer[0];
+            var viewToRemove = m_Orders[id];
+            m_OrdersViews.Remove(viewToRemove);
+        }
+
+        private OrderModel generateOrderModel()
+        {
+            eOrderState[] allStates = { eOrderState.Pending, eOrderState.Started, eOrderState.Done };
+            var items = generateOrderItemModels(m_Length + 100);
+            var orderModel = OrderBuilder.GenerateOrder(m_Length, m_Length + 100
+                , allStates[m_Length % 3], items);
+            m_Length++;
+            return orderModel;
+        }
+        private List<OrderItemModel> generateOrderItemModels(int i_TableNumber)
```

Comment:
This entire method, that is only about generating data from scratch, can and should be moved outside this class to be a static method in a utils folder